sdk/src/inflation.rs

```diff
@@ -69,6 +69,17 @@ impl Inflation {
         Self::new_fixed(0.0001) // 0.01% inflation
     }
 
+    pub fn full() -> Self {
+        Self {
+            initial: DEFAULT_INITIAL,
+            terminal: DEFAULT_TERMINAL,
+            taper: DEFAULT_TAPER,
```

Comment:
@CriesofCarrots @ericlwilliams are we sure these are the final inflation parameters, maybe?: https://github.com/solana-labs/solana/blob/a1b16a583489bd6cd3dc71f2924d0fe2563d10e4/sdk/src/inflation.rs#L25-L27

Reply:
I believe so: https://forums.solana.com/t/final-inflation-parameter-proposal-and-steps-to-implementation/977
But I certainly wouldn't mind @ericlwilliams 's eyes on this one more time before merge.

Reply:
Yes! as long as the `DEFAULT_TAPER` is implemented as negative in the calcs